scss/_variables.scss

```diff
@@ -24,3 +24,10 @@ $threeDots: url('data:image/svg+xml;utf8,<svg width="120" height="30" viewBox="0
 
 // Active loader
 $loader: $threeDots !default;
+
+// Icons
+$icon-correct: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAVCAMAAABxCz6aAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAA/1BMVEUW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22gW22j///80PlZcAAAAU3RSTlMAEi0+KgZ71/WwR1LrtRN5/twkW9QPHvOblcXl/AOdAVWCOae7xGP6dsOvBK5p3wq4pfSNysAzgQ6RvwKGeoyJk/kdFe7Q0xtB2Jxgt/jhkhciFMrrSWEAAAABYktHRFTkA4ilAAAAB3RJTUUH5AITFjoRLt95bAAAAN9JREFUGNNdkelSwkAQhDtCBCHKJYiSGJBDlCMqqCiXAnIqKvP+7+JkycLG/jHT9VXtTE0v4Ek7CAR1qNIOQ8QKH0X2LGqQ1PGJx2Jx2iuRFCylMqLTNLPMGfmVZXiugosckZmEZSjs0s5zLeBKYUVLL3Ero8L1unrjstuaVXe7gQbXpu0wvdOte/HgAe4QaiHdfnTs7HZKE0+iP0Pr4MUb/YpuTpgq0JP7+sBg64Zvkr2PgPHk30UtkZH/9g9HJNIwFTadednNFxKZy84u5drq84vR+jvo+xDYP78b6f8ACXhWBwGugsYAAAAldEVYdGRhdGU6Y3JlYXRlADIwMjAtMDItMjBUMDU6NTg6MTctMDc6MDCROErVAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDIwLTAyLTIwVDA1OjU4OjE3LTA3OjAw4GXyaQAAAABJRU5ErkJggg==') !default;
```

Comment:
Should we move all of these (including loaders that are already in this file) to a separate file and then include it here?

This is getting kind of messy.